content/docs/import/import-from-postgres.md

```diff
@@ -149,13 +149,13 @@ The `pg_dump` and `pg_restore` commands provide numerous advanced options, some
 - `-Z`: Defines the compression level to use when using a compressible format. 0 means no compression, while 9 means maximum compression. In general, we recommend a setting of 1. A higher compression level slows the dump and restore process but also uses less disk space.
 - `--lock-wait-timeout=20s`: Error out early in the dump process instead of waiting for an unknown amount of time if there is lock contention.
 Do not wait forever to acquire shared table locks at the beginning of the dump. Instead fail if unable to lock a table within the specified timeout.`
-- `-j <njobs>`: Consider this option for medium (>1 GB) and large databases (>5) to dump tables in parallel. Set `<njobs>` to the number of available CPUs. Refer to the [pg_dump](https://www.postgresql.org/docs/current/app-pgdump.html) documentation for more information. In Neon, this option only make sense for Pro plan users who can configure computes with >1 vCPU.
+- `-j <njobs>`: Consider this option for medium (>1 GiB) and large databases (>5) to dump tables in parallel. Set `<njobs>` to the number of available CPUs. Refer to the [pg_dump](https://www.postgresql.org/docs/current/app-pgdump.html) documentation for more information. In Neon, this option only make sense for Pro plan users who can configure computes with >1 vCPU.
 - `--no-blobs`: Excludes large objects from your dump. See [Data migration notes](#data-migration-notes).
 
 ### pg_restore options
 
 - `-c --if-exists`: Drop database objects before creating them if they already exist. If you had a failed migration, you can use these options to drop objects created by the previous migration to avoid errors when retrying the migration.
-- `-j <njobs>`: Consider this option for medium (>1 GB) and large databases (>5) to run the restore process in parallel. Set `<njobs>` to the number of available vCPUs. Refer to the [pg_dump](https://www.postgresql.org/docs/current/app-pgdump.html) documentation for more information. In Neon, this option only makes sense for Pro plan users who can configure computes with >1 vCPU. It cannot be used together with `--single-transaction`.
+- `-j <njobs>`: Consider this option for medium (>1 GiB) and large databases (>5) to run the restore process in parallel. Set `<njobs>` to the number of available vCPUs. Refer to the [pg_dump](https://www.postgresql.org/docs/current/app-pgdump.html) documentation for more information. In Neon, this option only makes sense for Pro plan users who can configure computes with >1 vCPU. It cannot be used together with `--single-transaction`.
```

Comment:
```suggestion
- `-j <njobs>`: Consider this option for medium (>1 GiB) and large databases (>5 GiB) to run the restore process in parallel. Set `<njobs>` to the number of available vCPUs. Refer to the [pg_dump](https://www.postgresql.org/docs/current/app-pgdump.html) documentation for more information. In Neon, this option only makes sense for Pro plan users who can configure computes with >1 vCPU. It cannot be used together with `--single-transaction`.
```

Reply:
FWIW i don't think it's critical to have GiB here.
the size the user will be getting from other systems, not neon

up to docs team to decide

Reply:
removed specific sizes